package/kernel/linux/modules/netdevices.mk

```diff
@@ -1457,3 +1457,18 @@ define KernelPackage/mhi-wwan-mbim/description
 endef
 
 $(eval $(call KernelPackage,mhi-wwan-mbim))
+
+define KernelPackage/aqc
```

Comment:
Dont think that `aqc` is the correct name, kernel calls the driver `aqtion`

Reply:
Changes it to `aqtion` thanks.

Reply:
For clarity **`atlantic`** was the correct value and `aqtion` would not load